app/views/layouts/application.html.erb

```diff
@@ -1,90 +1,95 @@
 <!DOCTYPE html>
 <html>
-  <head>
```

Comment:
I think indentation here looks weird, `head` need to be nested inside `html` tag.